ihatemoney/web.py

```diff
@@ -447,17 +448,45 @@ def import_project(file, project):
     json_file = json.load(file)
 
     # Check if JSON is correct
-    attr = ["what", "payer_name", "payer_weight", "amount", "date", "owers"]
+    attr = ["what", "payer_name", "payer_weight", "amount", "currency", "date", "owers"]
     attr.sort()
+    currencies = set()
     for e in json_file:
+        # Add compatibility with versions < 5 that did not have currencies
+        if "currency" not in e:
+            e["currency"] = CurrencyConverter.no_currency
+        # Empty currency should be converted to "XXX"
+        if e["currency"] == "":
+            e["currency"] = CurrencyConverter.no_currency
```

Comment:
```suggestion
        # Add compatibility with versions < 5 that did not have currencies
        # Empty currency should be converted to "XXX"
        if e.get("currency", "") == "":
            e["currency"] = CurrencyConverter.no_currency
```

Just to have a better readability